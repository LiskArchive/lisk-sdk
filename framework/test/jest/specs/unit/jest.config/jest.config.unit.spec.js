const config = require('../../../config/unit/jest.config');

describe('config/unit/jest.config.js', () => {
	it('unit test config must match to the snapshot.', () => {
		expect(config).toMatchSnapshot();
	});
});
