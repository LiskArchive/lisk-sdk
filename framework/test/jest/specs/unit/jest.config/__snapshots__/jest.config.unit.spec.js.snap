// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config/unit/jest.config.js unit test config must match to the snapshot. 1`] = `
Object {
  "collectCoverage": true,
  "collectCoverageFrom": Array [
    "framework/src/controller/**",
  ],
  "coverageDirectory": ".coverage/unit",
  "resetModules": true,
  "restoreMocks": true,
  "rootDir": "../../../../../",
  "setupFilesAfterEnv": Array [
    "<rootDir>/framework/test/jest/config/setup.js",
  ],
  "testMatch": Array [
    "<rootDir>/framework/test/jest/specs/unit/**/*.(spec|test).js",
  ],
  "verbose": true,
}
`;
